common/app/common/configuration.scala

```diff
@@ -474,6 +474,21 @@ class GuardianConfiguration extends Logging {
     lazy val config = configuration.getMandatoryStringProperty("front.config")
   }
 
+  object targeting {
+    lazy val campaignsUrl = configuration.getStringProperty("targeting.campaignsUrl")
+
+    // Total number of rules allowed per campaign, any campaigns with more than one rule will be filtered
+    lazy val ruleLimit: Int =
+      Try(configuration.getStringProperty("targeting.ruleLimit").get.toInt)
```

Comment:
Thanks! I've added that in my local branch, just trying to figure out why the tests are failing :)